packages/slice-machine/components/ScreenshotChangesModal/index.tsx

```diff
@@ -200,11 +203,20 @@ const ScreenshotChangesModal = ({
               flexGrow: 99999,
               flexBasis: 0,
               minWidth: 320,
+              maxHeight: "90%",
             }}
           >
-            SliceID: {variationSelector.sliceID}
-            <br />
-            VariationID: {variationSelector.variationID}
+            {(() => {
+              const slice = slices.find(
```

Comment:
you could use filter and map here, or reduce and map, that way it doesn't matter if it's undefined or not :)